spec/controllers/api/events_controller_spec.rb

```diff
@@ -144,7 +144,7 @@ def overridden_params(new_params)
 
       context "when the project id refers to a non-legacy migrated project" do
         let(:unknown_legacy_project) do
-          overridden_params(project_id: project.id)
+          overridden_params(project_id: 9999)
```

Comment:
Not sure why this needs to be change along with the project id as the created factory project id should not match the config => zoo home project id from the let(:project) block above in prev work. 

We could look at ensuring this via the 
``` ruby
let(:zoo_home_project_id) { (10..20).to_a.sample }
#could become
let(:zoo_home_project_id) { project.id + 1 }
```
The lookups for `event_params_check` calls the `legacy_zoo_project` method to find the legacy project via the config.